### ISSUE TITLE
1.1.0

### ISSUE THREAD

**Issue by ryust** [Blogcat/Android-ExpandableTextView#22]
Any idea when version 1.1.0 will be released? Or is the library dying off?

**Comment by Cliffus** [comment]
Hi 

the library isn't dead yet, but I'l have to admit I didn't put a lot of effort in it the last couple of months.

Which features are you mostly interested in?

Thanks!

**Comment by ryust** [comment]
There were some issues I encountered. It would be good if ExpandableTextView extended AppCompatTextView, but I'm able to do without it. Also found out that the ellipse doesn't show if the text was made selectable so dynamically switching it on/off based on collapse state would help. My main problem is the inability to get the text to collapse once it has been expanded... thought that was corrected in 1.1.0 release.

**Comment by Cliffus** [comment]
Thanks for your suggestions!

Can you make for each issue / feature request a ticket? I'll try to make some time to update this library to version 1.1.0.

Can you also provide me an example for the issue where the textview doesn't collapse anymore. Thanks a bunch!

**Comment by Cliffus** [comment]
Hi Ron

Did you already find some time to make some example code so I can reproduce and fix your issue?

Thanks a lot!
Cliff

**Comment by ryust** [comment]
No, I just set my app to not collapse the text once it was expanded.

**Comment by Cliffus** [comment]
can you than maybe share some code with me, so I can investigate this issue?

**Comment by ryust** [comment]
Sorry, but I don't have a simple test case. The code is part of a larger framework so there's nothing easily shareable. I gave up trying to figure it out.

**Comment by Cliffus** [comment]
Hi Ron

I understand you can't share code of your clients project, but can you at least say in which ViewGroup / layout setup you are using the ExpandableTextView?

I would appreciate it a lot!

**Comment by ryust** [comment]
It's immediate parent is a vertical LinearLayout... which is in a TableView... which is in another vertical LinearLayout. Maybe the TableView is a culprit?

**Comment by Cliffus** [comment]
thanks for the information! I'll take a look at it
